resource-managers/kubernetes/integration-tests/tests/pyfiles.py

```diff
@@ -35,4 +36,11 @@
     # Begin of Python container checks
     version_check(sys.argv[1], 2 if sys.argv[1] == "python" else 3)
 
+    # Check python executable at executors
+    spark.catalog.registerFunction("getSysVer",
+                                   lambda: "%d.%d" % sys.version_info[:2], StringType())
+    [row] = spark.sql("SELECT getSysVer()").collect()
+    driverVersion = "%d.%d" % sys.version_info[:2]
```

Comment:
shall we stick to `driver_version` name? `getSysVer` -> `get_sys_ver` too

Reply:
Yes.